src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -698,7 +698,7 @@ public AppPage verifyHtml(String filePath) {
             filePath = TestProperties.TEST_PAGES_FOLDER + filePath;
         }
         String actual = getPageSource();
-        
+        actual = processPageSourceForGodMode(actual);
```

Comment:
Since the url is different in dev and staging, we have to substitute the url to something standard before we can compare it with the expected html. So we convert "http://localhost:8888" and "https://<version>-dot-teammatesv4.appspot.com" to "${web.url}" before comparing. To do this, the general way would be to simply pass the actual html through the "processPageSourceForGodMode" method.